app/components/post_textbox/index.js

```diff
@@ -41,6 +43,7 @@ function mapStateToProps(state, ownProps) {
         channelId: ownProps.channelId || (currentChannel ? currentChannel.id : ''),
         canUploadFiles: canUploadFilesOnMobile(state),
         channelIsLoading: state.views.channel.loading,
+        checkMessageLength: !isMinimumServerVersion(Client4.getServerVersion(), 4, 9),
```

Comment:
I couldn't find this is used elsewhere.

Reply:
Hmm, that is for the long post dunno why is it here, I’ll take a look